tests/CMakeLists.txt

```diff
@@ -99,6 +99,9 @@ if(AVIF_ENABLE_GTEST)
     add_test(NAME avifmetadatatest COMMAND avifmetadatatest ${CMAKE_CURRENT_SOURCE_DIR}/data/)
 
     add_executable(avifrgbtoyuvtest gtest/avifrgbtoyuvtest.cc)
+    if(libsharpyuv_FOUND)
+        target_compile_definitions(avifrgbtoyuvtest PRIVATE -DAVIF_LIBSHARPYUV_ENABLED=1)
+    endif()
```

Comment:
What about the following, similar to line 123:

```cmake
else()
    message(STATUS "Some tests are disabled because libsharpyuv is unavailable.")
```